### DISCUSSION TITLE
请问大佬，语速控制是如何实现的呢？

### DISCUSSION THREAD

**Discussion by KylinMountain** [lenML/Speech-AI-Forge#85] [问答 | Q&A]
### 阅读 README.md 和 dependencies.md

- [X] 我已经阅读过 README.md 和 dependencies.md 文件

### 检索 issue 和 discussion

- [X] 我已经确认之前没有 issue 或 discussion 涉及此 BUG

### 检查 Forge 版本

- [X] 我已经确认问题发生在最新代码或稳定版本中

### 你的issues

我用原版的ChatTTS，使用Prompt中speed控制没有什么效果，我自己使用调整帧率的方式来调节播放速度，发现人声音都变了。。。
但我测试大佬的Forge版本，效果不错，请教一下如何实现呢？

**Comment by zhzLuke96** [comment]
使用 pyrubberband 这个库
